sdk/resources/mgmt/src/main/java/com/azure/resourcemanager/resources/fluentcore/AzureServiceClient.java

```diff
@@ -83,24 +70,11 @@ public SerializerAdapter getSerializerAdapter() {
      * @return the default client context.
      */
     public Context getContext() {
-        Context context = new Context("java.version", JAVA_VERSION);
-        if (!CoreUtils.isNullOrEmpty(OS_NAME)) {
-            context = context.addData("os.name", OS_NAME);
-        }
-        if (!CoreUtils.isNullOrEmpty(OS_VERSION)) {
-            context = context.addData("os.version", OS_VERSION);
-        }
         if (sdkName == null) {
-            String packageName = this.getClass().getPackage().getName();
-            if (packageName.endsWith(".models")) {
-                sdkName = packageName.substring(0, packageName.length() - ".models".length());
-            } else {
-                sdkName = packageName;
-            }
+            sdkName = this.getClass().getPackage().getName();
         }
-        context = context.addData("Sdk-Name", sdkName);
-        context = context.addData("Sdk-Version", SDK_VERSION);
-        return context;
+        return new Context("Sdk-Name", sdkName)
+            .addData("Sdk-Version", SDK_VERSION);
```

Comment:
why sdk version hardcoded?

Reply:
there might be other approach. not investigated yet.

Reply:
got it. let's fix it before GA.

Reply:
already fixed in this PR.

now the flow is:
1. maven package the project, one step it would populate `azure.properties` with its current `project.version`.
2. jar now has this `azure.properties` packaged
3. runtime, code read version from `azure.properties`